docs/content/deployment/run-monitoring.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Run Monitoring | Dagster
+description: Features for manging run failures in Dagster.
+---
+
+# Run Monitoring
+
+Dagster can detect hanging runs and restart crashed run workers. Run monitoring is currently only supported with:
+
+- `K8sRunLauncher`
+- `CeleryK8sRunLauncher`
+- `DockerRunLauncher`
+
+These features require running the Dagster Daemon, and enabling run monitoring in the Dagster Instance:
+
+```yaml file=/deploying/dagster_instance/dagster.yaml startafter=start_run_monitoring endbefore=end_run_monitoring
+# Opt in to the experimental Monitoring Daemon
+run_monitoring:
+  enabled: true
+  # values below are the defaults, and don't need to be specified execpt to override them
+  start_timeout_seconds: 180
+  max_resume_run_attempts: 3 # experimental if above 0
+  poll_interval_seconds: 120
+```
+
+## Run start timeouts
+
+After a RunLauncher launches a run, the run stays in STARTING status until the run worker spins up and marks the run as STARTED. In the event that some failure causes the run worker to not spin up, the run might be stuck in STARTING status. The `start_timeout_seconds` offers a time limit for how long runs can hang in this state before being marked as failed.
+
+## Detecting run worker crashes
+
+If a run worker crashes, the run it's manging can hang. The monitoring daemon can run health checks on run workers for all active runs to detect this. If a failed run worker is detected (e.g. by the K8s Job having a non-zero exit code), the run is either marked as failed or resumed (see below).
+
+## Resuming runs after run worker crashes (Experimental)
+
+This feature is experimental and currently only supported when using:
+
+- `K8sRunLauncher` with the `k8s_job_executor`
+- `DockerRunLauncher` with the `docker_executor`
+
+It's possible for a run worker process to crash during a run. This can happen for a variety of reasons (the host it's running on could go down, it could OOM, etc.). Without the monitoring daemon, there are two possible outcomes, neither desirable:
+
+- If the run worker was able to catch the interrupt, it will mark the run as failed
+- If the run worker goes down without a grace period, the run could be left hanging in STARTED status
+
+The monitoring daemon handles these by performing health checks on the run workers. If a failure is detected, the daemon can launch a new run worker which resumes execution of the existing run. This can give the appearance that the failure never occured (although it will be in the event log), and the run will continue to completion. If the run worker continues to crash, the daemon will mark the run as failed after the configured numbe of attempts.
```

Comment:
"This can give the appearance that the failure never occurred" sounds like something bad. maybe

"The fact that the run was resumed after a crash will be shown in the event log, and the run will continue to completion"